configuration/uart.md

```diff
@@ -1,46 +1,61 @@
  
-By default, on Raspberry Pis equipped with the wireless/Bluetooth module (Raspberry Pi 3 and Raspberry Pi Zero W), the PL011 UART is connected to the Bluetooth module, while the mini UART is used as the primary UART and will have a Linux console on it. On all other models, the PL011 is used as the primary UART. 
+By default, on Raspberry Pis equipped with the combined wireless LAN/Bluetooth module, the PL011 is connected to the Bluetooth module, while the mini UART is used as the primary UART and will have a Linux console on it. On all other models, the PL011 is used as the primary UART. 
 
-In Linux device terms, by default, /dev/ttyS0 refers to the mini UART, and /dev/ttyAMA0 refers to the PL011. The primary UART is the one assigned to the Linux console, which depends on the Raspberry Pi model as described above. There are also symlinks: /dev/serial0, which always refers to the primary UART (if enabled), and /dev/serial1, which similarly always refers to the secondary UART (if enabled).
+The following table summarises the default configuration of the UARTs:
 
-## Mini UART and CPU core frequency
+| Model | mini UART | PL011 |
+|-------|-----------|-------|
+| Raspberry Pi Zero | secondary | primary |
+| Raspberry Pi Zero W | primary (disabled) | secondary (Bluetooth) |
+| Raspberry Pi 1 | secondary | primary |
+| Raspberry Pi 2 | secondary | primary |
+| Raspberry Pi 3 | primary (disabled) | secondary (Bluetooth) |
+| Raspberry Pi 4 | primary (disabled) | secondary (Bluetooth) |
 
-The baud rate of the mini UART is linked to the core frequency of the VPU on the VC4 GPU. This means that, as the VPU frequency governor varies the core frequency, the baud rate of the mini UART also changes. This makes the mini UART of limited use in the default state. By default, if the mini UART is selected for use as the primary UART, it will be disabled. To enable it, add `enable_uart=1` to config.txt. This will also fix the core frequency to 250MHz (unless `force_turbo` is set, when it will be fixed to the VPU turbo frequency). When the mini UART is not the primary UART, for example you are using it to connect to the Bluetooth controller, you must add `core_freq=250` to config.txt, otherwise the mini UART will not work.
+Linux devices on Raspbian:
 
-The default value of the `enable_uart` flag depends on the actual roles of the UARTs, so that if ttyAMA0 is assigned to the Bluetooth module, `enable_uart` defaults to 0. If the mini UART is assigned to the Bluetooth module, then `enable_uart` defaults to 1. Note that if the UARTs are reassigned using a Device Tree Overlay (see below), `enable_uart` defaults will still obey this rule.
+| Linux device | Description |
+|--------------|-------------|
+|`/dev/ttyS0`  |mini UART    |
+|`/dev/ttyAMA0`|PL011        |
+|`/dev/serial0` |primary UART (Linux console) |
+|`/dev/serial1` |secondary UART |
```

Comment:
It's probably worth explaining that the `/dev/serial` devices are actually symlinks to the other devices? (IIRC the symlinks are auto-adjusted based on the model of Pi and devicetree-settings)

Reply:
I've added mention of them being symlinks. The page already mentioned the primary UART being on the GPIO header.